### PR TITLE
Add embed type and autoplay audio options for Vine

### LINKED ISSUE #83: Add shortcode for Vine
Also needs reversal for this:

```
<iframe src="https://vine.co/v/eO7aFmWXeYg/embed/simple" width="600" height="600" frameborder="0"></iframe>
```

### PR DESCRIPTION
**Pull request by danielbachhuber**
Fixes #83

### CODE REVIEW

**Comment by danielbachhuber** [comment]
@goldenapples #reviewmerge

File: phpcs.ruleset.xml [hunk 1]
```diff
@@ -5,6 +5,7 @@
 	<rule ref="WordPress-VIP">
 		<exclude name="WordPress.CSRF.NonceVerification" />
 		<exclude name="WordPress.WhiteSpace.ControlStructureSpacing" />
+		<exclude name="WordPress.VIP.SlowDBQuery" />
```
Comment (goldenapples) [inline phpcs.ruleset.xml]: Why did this have to change here?

File: inc/shortcodes/class-vine.php [hunk 2]
```diff
@@ -47,11 +71,20 @@ public static function callback( $attrs, $content = '' ) {
...
+			$type = 'simple';
+		}
+
 		// ID is always the second part to the path
 		$path = parse_url( $attrs['url'], PHP_URL_PATH );
 		$parts = explode( '/', trim( $path, '/' ) );
 		$embed_id = $parts[1];
-		$embed_url = 'https://vine.co/v/' . $embed_id . '/embed/simple';
+		$embed_url = 'https://vine.co/v/' . $embed_id . '/embed/' . $type;
+		if ( ! empty( $attrs['autoplay'] ) ) {
```
Comment (goldenapples) [inline inc/shortcodes/class-vine.php]: Should also check for `"0"` value.

File: phpcs.ruleset.xml (hunk 1 above)
Comment (danielbachhuber) [inline phpcs.ruleset.xml]: No idea https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/issues/471

**Comment by goldenapples** [comment]
Saw one potential bug - looks like creating the shortcode using the UI will always result in autoplaying audio. Otherwise, it looks good...

File: inc/shortcodes/class-vine.php (hunk 2 above)
Comment (danielbachhuber) [inline inc/shortcodes/class-vine.php]: Why?

```
wp> $autoplay = '0';
string(1) "0"
wp> ! empty( $autoplay )
bool(false)
```

#phpfolife

Comment (goldenapples) [inline inc/shortcodes/class-vine.php]: whaaaaat...? ok. fine, then.

**Pull request merged**